Build failure w/ base-4.10.1.0

`rattletrap-0.5.0` advertises compatibility with `base >=4.10.1 && <4.12`, however GHC disagrees

```
Configuring library for rattletrap-5.0.0..
Preprocessing library for rattletrap-5.0.0..
Building library for rattletrap-5.0.0..
[  1 of 227] Compiling Paths_rattletrap ( /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/autogen/Paths_rattletrap.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Paths_rattletrap.o )
[  2 of 227] Compiling Rattletrap.Type.Bitstream ( library/Rattletrap/Type/Bitstream.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Bitstream.o )
[  3 of 227] Compiling Rattletrap.Encode.Bitstream ( library/Rattletrap/Encode/Bitstream.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Bitstream.o )
[  4 of 227] Compiling Rattletrap.Type.Common ( library/Rattletrap/Type/Common.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Common.o )
[  5 of 227] Compiling Rattletrap.Type.BooleanAttribute ( library/Rattletrap/Type/BooleanAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/BooleanAttribute.o )
[  6 of 227] Compiling Rattletrap.Encode.BooleanAttribute ( library/Rattletrap/Encode/BooleanAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/BooleanAttribute.o )
[  7 of 227] Compiling Rattletrap.Type.AttributeType ( library/Rattletrap/Type/AttributeType.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/AttributeType.o )
[  8 of 227] Compiling Rattletrap.Data  ( library/Rattletrap/Data.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Data.o )
[  9 of 227] Compiling Rattletrap.Type.CompressedWord ( library/Rattletrap/Type/CompressedWord.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/CompressedWord.o )
[ 10 of 227] Compiling Rattletrap.Encode.CompressedWord ( library/Rattletrap/Encode/CompressedWord.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/CompressedWord.o )
[ 11 of 227] Compiling Rattletrap.Type.CompressedWordVector ( library/Rattletrap/Type/CompressedWordVector.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/CompressedWordVector.o )
[ 12 of 227] Compiling Rattletrap.Encode.CompressedWordVector ( library/Rattletrap/Encode/CompressedWordVector.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/CompressedWordVector.o )
[ 13 of 227] Compiling Rattletrap.Type.DestroyedReplication ( library/Rattletrap/Type/DestroyedReplication.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/DestroyedReplication.o )
[ 14 of 227] Compiling Rattletrap.Encode.DestroyedReplication ( library/Rattletrap/Encode/DestroyedReplication.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/DestroyedReplication.o )
[ 15 of 227] Compiling Rattletrap.Type.EnumAttribute ( library/Rattletrap/Type/EnumAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/EnumAttribute.o )
[ 16 of 227] Compiling Rattletrap.Encode.EnumAttribute ( library/Rattletrap/Encode/EnumAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/EnumAttribute.o )
[ 17 of 227] Compiling Rattletrap.Type.Float32le ( library/Rattletrap/Type/Float32le.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Float32le.o )
[ 18 of 227] Compiling Rattletrap.Type.CamSettingsAttribute ( library/Rattletrap/Type/CamSettingsAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/CamSettingsAttribute.o )
[ 19 of 227] Compiling Rattletrap.Type.FloatAttribute ( library/Rattletrap/Type/FloatAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/FloatAttribute.o )
[ 20 of 227] Compiling Rattletrap.Type.GameModeAttribute ( library/Rattletrap/Type/GameModeAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/GameModeAttribute.o )
[ 21 of 227] Compiling Rattletrap.Encode.GameModeAttribute ( library/Rattletrap/Encode/GameModeAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/GameModeAttribute.o )
[ 22 of 227] Compiling Rattletrap.Type.Int32le ( library/Rattletrap/Type/Int32le.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Int32le.o )
[ 23 of 227] Compiling Rattletrap.Type.FlaggedIntAttribute ( library/Rattletrap/Type/FlaggedIntAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/FlaggedIntAttribute.o )
[ 24 of 227] Compiling Rattletrap.Type.Int8le ( library/Rattletrap/Type/Int8le.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Int8le.o )
[ 25 of 227] Compiling Rattletrap.Type.Int8Vector ( library/Rattletrap/Type/Int8Vector.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Int8Vector.o )
[ 26 of 227] Compiling Rattletrap.Type.IntAttribute ( library/Rattletrap/Type/IntAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/IntAttribute.o )
[ 27 of 227] Compiling Rattletrap.Type.List ( library/Rattletrap/Type/List.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/List.o )
[ 28 of 227] Compiling Rattletrap.Type.PlayerHistoryKeyAttribute ( library/Rattletrap/Type/PlayerHistoryKeyAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/PlayerHistoryKeyAttribute.o )
[ 29 of 227] Compiling Rattletrap.Encode.PlayerHistoryKeyAttribute ( library/Rattletrap/Encode/PlayerHistoryKeyAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/PlayerHistoryKeyAttribute.o )
[ 30 of 227] Compiling Rattletrap.Type.Quaternion ( library/Rattletrap/Type/Quaternion.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Quaternion.o )

library/Rattletrap/Type/Quaternion.hs: warning:
    Could not specialise imported function ‘GHC.Float.$w$cround’
      when specialising ‘GHC.Float.$fRealFracDouble_$cround’
    Probable fix: add INLINABLE pragma on ‘GHC.Float.$w$cround’
[ 31 of 227] Compiling Rattletrap.Encode.Quaternion ( library/Rattletrap/Encode/Quaternion.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Quaternion.o )
[ 32 of 227] Compiling Rattletrap.Type.Rotation ( library/Rattletrap/Type/Rotation.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Rotation.o )
[ 33 of 227] Compiling Rattletrap.Encode.Rotation ( library/Rattletrap/Encode/Rotation.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Rotation.o )
[ 34 of 227] Compiling Rattletrap.Type.StatEventAttribute ( library/Rattletrap/Type/StatEventAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/StatEventAttribute.o )
[ 35 of 227] Compiling Rattletrap.Type.Str ( library/Rattletrap/Type/Str.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Str.o )
[ 36 of 227] Compiling Rattletrap.Type.Dictionary ( library/Rattletrap/Type/Dictionary.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Dictionary.o )
[ 37 of 227] Compiling Rattletrap.Type.StringAttribute ( library/Rattletrap/Type/StringAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/StringAttribute.o )
[ 38 of 227] Compiling Rattletrap.Type.Vector ( library/Rattletrap/Type/Vector.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Vector.o )
[ 39 of 227] Compiling Rattletrap.Type.RigidBodyStateAttribute ( library/Rattletrap/Type/RigidBodyStateAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/RigidBodyStateAttribute.o )
[ 40 of 227] Compiling Rattletrap.Type.LocationAttribute ( library/Rattletrap/Type/LocationAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/LocationAttribute.o )
[ 41 of 227] Compiling Rattletrap.Type.Initialization ( library/Rattletrap/Type/Initialization.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Initialization.o )
[ 42 of 227] Compiling Rattletrap.Type.ExplosionAttribute ( library/Rattletrap/Type/ExplosionAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/ExplosionAttribute.o )
[ 43 of 227] Compiling Rattletrap.Type.ExtendedExplosionAttribute ( library/Rattletrap/Type/ExtendedExplosionAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/ExtendedExplosionAttribute.o )
[ 44 of 227] Compiling Rattletrap.Encode.Vector ( library/Rattletrap/Encode/Vector.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Vector.o )

library/Rattletrap/Encode/Vector.hs: warning:
    Could not specialise imported function ‘GHC.Float.$w$cround1’
      when specialising ‘GHC.Float.$fRealFracFloat_$cround’
    Probable fix: add INLINABLE pragma on ‘GHC.Float.$w$cround1’
[ 45 of 227] Compiling Rattletrap.Encode.RigidBodyStateAttribute ( library/Rattletrap/Encode/RigidBodyStateAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/RigidBodyStateAttribute.o )
[ 46 of 227] Compiling Rattletrap.Encode.LocationAttribute ( library/Rattletrap/Encode/LocationAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/LocationAttribute.o )
[ 47 of 227] Compiling Rattletrap.Type.WeldedInfoAttribute ( library/Rattletrap/Type/WeldedInfoAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/WeldedInfoAttribute.o )
[ 48 of 227] Compiling Rattletrap.Type.Word32le ( library/Rattletrap/Type/Word32le.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Word32le.o )
[ 49 of 227] Compiling Rattletrap.Type.TitleAttribute ( library/Rattletrap/Type/TitleAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/TitleAttribute.o )
[ 50 of 227] Compiling Rattletrap.Type.SpawnedReplication ( library/Rattletrap/Type/SpawnedReplication.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/SpawnedReplication.o )
[ 51 of 227] Compiling Rattletrap.Type.Section ( library/Rattletrap/Type/Section.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Section.o )
[ 52 of 227] Compiling Rattletrap.Type.ProductAttribute ( library/Rattletrap/Type/ProductAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/ProductAttribute.o )
[ 53 of 227] Compiling Rattletrap.Type.LoadoutOnlineAttribute ( library/Rattletrap/Type/LoadoutOnlineAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/LoadoutOnlineAttribute.o )
[ 54 of 227] Compiling Rattletrap.Type.LoadoutsOnlineAttribute ( library/Rattletrap/Type/LoadoutsOnlineAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/LoadoutsOnlineAttribute.o )
[ 55 of 227] Compiling Rattletrap.Type.PrivateMatchSettingsAttribute ( library/Rattletrap/Type/PrivateMatchSettingsAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/PrivateMatchSettingsAttribute.o )
[ 56 of 227] Compiling Rattletrap.Type.PickupAttribute ( library/Rattletrap/Type/PickupAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/PickupAttribute.o )
[ 57 of 227] Compiling Rattletrap.Type.Message ( library/Rattletrap/Type/Message.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Message.o )
[ 58 of 227] Compiling Rattletrap.Type.Mark ( library/Rattletrap/Type/Mark.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Mark.o )
[ 59 of 227] Compiling Rattletrap.Type.KeyFrame ( library/Rattletrap/Type/KeyFrame.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/KeyFrame.o )
[ 60 of 227] Compiling Rattletrap.Type.DemolishAttribute ( library/Rattletrap/Type/DemolishAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/DemolishAttribute.o )
[ 61 of 227] Compiling Rattletrap.Type.ClassMapping ( library/Rattletrap/Type/ClassMapping.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/ClassMapping.o )
[ 62 of 227] Compiling Rattletrap.Type.AttributeMapping ( library/Rattletrap/Type/AttributeMapping.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/AttributeMapping.o )
[ 63 of 227] Compiling Rattletrap.Type.Cache ( library/Rattletrap/Type/Cache.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Cache.o )

library/Rattletrap/Type/Cache.hs: warning:
    Could not specialise imported function ‘Rattletrap.Type.List.$w$cshowsPrec’
      when specialising ‘Rattletrap.Type.List.$fShowList_$cshowsPrec’
    Probable fix: add INLINABLE pragma on ‘Rattletrap.Type.List.$w$cshowsPrec’
[ 64 of 227] Compiling Rattletrap.Type.ClassAttributeMap ( library/Rattletrap/Type/ClassAttributeMap.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/ClassAttributeMap.o )

library/Rattletrap/Type/ClassAttributeMap.hs: warning:
    Could not specialise imported function ‘Data.Map.Internal.$w$cshowsPrec’
      when specialising ‘Data.Map.Internal.$fShowMap_$cshowsPrec’
    Probable fix: add INLINABLE pragma on ‘Data.Map.Internal.$w$cshowsPrec’

library/Rattletrap/Type/ClassAttributeMap.hs: warning:
    Could not specialise imported function ‘Data.IntMap.Internal.$w$cshowsPrec’
      when specialising ‘Data.IntMap.Internal.$fShowIntMap_$cshowsPrec’
    Probable fix: add INLINABLE pragma on ‘Data.IntMap.Internal.$w$cshowsPrec’
[ 65 of 227] Compiling Rattletrap.Type.Word64le ( library/Rattletrap/Type/Word64le.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Word64le.o )
[ 66 of 227] Compiling Rattletrap.Type.RemoteId ( library/Rattletrap/Type/RemoteId.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/RemoteId.o )
[ 67 of 227] Compiling Rattletrap.Type.QWordAttribute ( library/Rattletrap/Type/QWordAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/QWordAttribute.o )
[ 68 of 227] Compiling Rattletrap.Type.Word8le ( library/Rattletrap/Type/Word8le.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Word8le.o )
[ 69 of 227] Compiling Rattletrap.Type.UniqueIdAttribute ( library/Rattletrap/Type/UniqueIdAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/UniqueIdAttribute.o )
[ 70 of 227] Compiling Rattletrap.Type.ReservationAttribute ( library/Rattletrap/Type/ReservationAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/ReservationAttribute.o )
[ 71 of 227] Compiling Rattletrap.Type.TeamPaintAttribute ( library/Rattletrap/Type/TeamPaintAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/TeamPaintAttribute.o )
[ 72 of 227] Compiling Rattletrap.Type.PropertyValue ( library/Rattletrap/Type/PropertyValue.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/PropertyValue.o )
[ 73 of 227] Compiling Rattletrap.Type.Property ( library/Rattletrap/Type/Property.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Property.o )
[ 74 of 227] Compiling Rattletrap.Type.Header ( library/Rattletrap/Type/Header.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Header.o )
[ 75 of 227] Compiling Rattletrap.Type.PartyLeaderAttribute ( library/Rattletrap/Type/PartyLeaderAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/PartyLeaderAttribute.o )

library/Rattletrap/Type/PartyLeaderAttribute.hs: warning:
    Could not specialise imported function ‘ghc-prim-0.5.1.1:GHC.Classes.$w$ccompare’
      when specialising ‘ghc-prim-0.5.1.1:GHC.Classes.$fOrd(,)_$ccompare’
    Probable fix: add INLINABLE pragma on ‘ghc-prim-0.5.1.1:GHC.Classes.$w$ccompare’

library/Rattletrap/Type/PartyLeaderAttribute.hs: warning:
    Could not specialise imported function ‘ghc-prim-0.5.1.1:GHC.Classes.$w$c==’
      when specialising ‘ghc-prim-0.5.1.1:GHC.Classes.$fEq(,)_$c==’
    Probable fix: add INLINABLE pragma on ‘ghc-prim-0.5.1.1:GHC.Classes.$w$c==’

library/Rattletrap/Type/PartyLeaderAttribute.hs: warning:
    Could not specialise imported function ‘ghc-prim-0.5.1.1:GHC.Classes.$w$c<’
      when specialising ‘ghc-prim-0.5.1.1:GHC.Classes.$fOrd(,)_$c<’
    Probable fix: add INLINABLE pragma on ‘ghc-prim-0.5.1.1:GHC.Classes.$w$c<’
[ 76 of 227] Compiling Rattletrap.Type.MusicStingerAttribute ( library/Rattletrap/Type/MusicStingerAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/MusicStingerAttribute.o )
[ 77 of 227] Compiling Rattletrap.Type.LoadoutAttribute ( library/Rattletrap/Type/LoadoutAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/LoadoutAttribute.o )
[ 78 of 227] Compiling Rattletrap.Type.LoadoutsAttribute ( library/Rattletrap/Type/LoadoutsAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/LoadoutsAttribute.o )
[ 79 of 227] Compiling Rattletrap.Type.DamageStateAttribute ( library/Rattletrap/Type/DamageStateAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/DamageStateAttribute.o )
[ 80 of 227] Compiling Rattletrap.Type.ClubColorsAttribute ( library/Rattletrap/Type/ClubColorsAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/ClubColorsAttribute.o )
[ 81 of 227] Compiling Rattletrap.Type.ByteAttribute ( library/Rattletrap/Type/ByteAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/ByteAttribute.o )
[ 82 of 227] Compiling Rattletrap.Type.AppliedDamageAttribute ( library/Rattletrap/Type/AppliedDamageAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/AppliedDamageAttribute.o )
[ 83 of 227] Compiling Rattletrap.Type.AttributeValue ( library/Rattletrap/Type/AttributeValue.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/AttributeValue.o )
[ 84 of 227] Compiling Rattletrap.Type.Attribute ( library/Rattletrap/Type/Attribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Attribute.o )
[ 85 of 227] Compiling Rattletrap.Type.UpdatedReplication ( library/Rattletrap/Type/UpdatedReplication.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/UpdatedReplication.o )
[ 86 of 227] Compiling Rattletrap.Type.ReplicationValue ( library/Rattletrap/Type/ReplicationValue.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/ReplicationValue.o )
[ 87 of 227] Compiling Rattletrap.Type.Replication ( library/Rattletrap/Type/Replication.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Replication.o )
[ 88 of 227] Compiling Rattletrap.Type.Frame ( library/Rattletrap/Type/Frame.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Frame.o )
[ 89 of 227] Compiling Rattletrap.Type.Content ( library/Rattletrap/Type/Content.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Content.o )

library/Rattletrap/Type/Content.hs: warning:
    Could not specialise imported function ‘Rattletrap.Type.List.$w$cshowsPrec’
      when specialising ‘Rattletrap.Type.List.$fShowList_$cshowsPrec’
    Probable fix: add INLINABLE pragma on ‘Rattletrap.Type.List.$w$cshowsPrec’
[ 90 of 227] Compiling Rattletrap.Type.Replay ( library/Rattletrap/Type/Replay.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Type/Replay.o )

library/Rattletrap/Type/Replay.hs: warning:
    Could not specialise imported function ‘Rattletrap.Type.Section.$w$ccompare’
      when specialising ‘Rattletrap.Type.Section.$fOrdSection_$ccompare’
    Probable fix: add INLINABLE pragma on ‘Rattletrap.Type.Section.$w$ccompare’

library/Rattletrap/Type/Replay.hs: warning:
    Could not specialise imported function ‘Rattletrap.Type.Section.$w$c==’
      when specialising ‘Rattletrap.Type.Section.$fEqSection_$c==’
    Probable fix: add INLINABLE pragma on ‘Rattletrap.Type.Section.$w$c==’

library/Rattletrap/Type/Replay.hs: warning:
    Could not specialise imported function ‘Rattletrap.Type.Section.$w$c<’
      when specialising ‘Rattletrap.Type.Section.$fOrdSection_$c<’
    Probable fix: add INLINABLE pragma on ‘Rattletrap.Type.Section.$w$c<’

library/Rattletrap/Type/Replay.hs: warning:
    Could not specialise imported function ‘Rattletrap.Type.Section.$w$cshowsPrec’
      when specialising ‘Rattletrap.Type.Section.$fShowSection_$cshowsPrec’
    Probable fix: add INLINABLE pragma on ‘Rattletrap.Type.Section.$w$cshowsPrec’

library/Rattletrap/Type/Replay.hs: warning:
    Could not specialise imported function ‘Rattletrap.Type.Section.$w$ctoJSON’
      when specialising ‘Rattletrap.Type.Section.$fToJSONSection_$ctoJSON’
    Probable fix: add INLINABLE pragma on ‘Rattletrap.Type.Section.$w$ctoJSON’
[ 91 of 227] Compiling Rattletrap.Utility.Bytes ( library/Rattletrap/Utility/Bytes.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Utility/Bytes.o )
[ 92 of 227] Compiling Rattletrap.Encode.Word8le ( library/Rattletrap/Encode/Word8le.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Word8le.o )
[ 93 of 227] Compiling Rattletrap.Encode.ClubColorsAttribute ( library/Rattletrap/Encode/ClubColorsAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/ClubColorsAttribute.o )
[ 94 of 227] Compiling Rattletrap.Encode.ByteAttribute ( library/Rattletrap/Encode/ByteAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/ByteAttribute.o )
[ 95 of 227] Compiling Rattletrap.Encode.Word64le ( library/Rattletrap/Encode/Word64le.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Word64le.o )
[ 96 of 227] Compiling Rattletrap.Encode.QWordAttribute ( library/Rattletrap/Encode/QWordAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/QWordAttribute.o )
[ 97 of 227] Compiling Rattletrap.Encode.Word32le ( library/Rattletrap/Encode/Word32le.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Word32le.o )
[ 98 of 227] Compiling Rattletrap.Encode.TitleAttribute ( library/Rattletrap/Encode/TitleAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/TitleAttribute.o )
[ 99 of 227] Compiling Rattletrap.Encode.TeamPaintAttribute ( library/Rattletrap/Encode/TeamPaintAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/TeamPaintAttribute.o )
[100 of 227] Compiling Rattletrap.Encode.ProductAttribute ( library/Rattletrap/Encode/ProductAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/ProductAttribute.o )
[101 of 227] Compiling Rattletrap.Encode.LoadoutOnlineAttribute ( library/Rattletrap/Encode/LoadoutOnlineAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/LoadoutOnlineAttribute.o )
[102 of 227] Compiling Rattletrap.Encode.LoadoutsOnlineAttribute ( library/Rattletrap/Encode/LoadoutsOnlineAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/LoadoutsOnlineAttribute.o )
[103 of 227] Compiling Rattletrap.Encode.PickupAttribute ( library/Rattletrap/Encode/PickupAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/PickupAttribute.o )
[104 of 227] Compiling Rattletrap.Encode.MusicStingerAttribute ( library/Rattletrap/Encode/MusicStingerAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/MusicStingerAttribute.o )
[105 of 227] Compiling Rattletrap.Encode.LoadoutAttribute ( library/Rattletrap/Encode/LoadoutAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/LoadoutAttribute.o )
[106 of 227] Compiling Rattletrap.Encode.LoadoutsAttribute ( library/Rattletrap/Encode/LoadoutsAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/LoadoutsAttribute.o )
[107 of 227] Compiling Rattletrap.Encode.List ( library/Rattletrap/Encode/List.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/List.o )
[108 of 227] Compiling Rattletrap.Encode.DemolishAttribute ( library/Rattletrap/Encode/DemolishAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/DemolishAttribute.o )
[109 of 227] Compiling Rattletrap.Encode.AttributeMapping ( library/Rattletrap/Encode/AttributeMapping.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/AttributeMapping.o )
[110 of 227] Compiling Rattletrap.Encode.Cache ( library/Rattletrap/Encode/Cache.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Cache.o )
[111 of 227] Compiling Rattletrap.Encode.RemoteId ( library/Rattletrap/Encode/RemoteId.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/RemoteId.o )
[112 of 227] Compiling Rattletrap.Encode.UniqueIdAttribute ( library/Rattletrap/Encode/UniqueIdAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/UniqueIdAttribute.o )
[113 of 227] Compiling Rattletrap.Encode.PartyLeaderAttribute ( library/Rattletrap/Encode/PartyLeaderAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/PartyLeaderAttribute.o )
[114 of 227] Compiling Rattletrap.Encode.Int8le ( library/Rattletrap/Encode/Int8le.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Int8le.o )
[115 of 227] Compiling Rattletrap.Encode.Int8Vector ( library/Rattletrap/Encode/Int8Vector.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Int8Vector.o )
[116 of 227] Compiling Rattletrap.Encode.Initialization ( library/Rattletrap/Encode/Initialization.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Initialization.o )
[117 of 227] Compiling Rattletrap.Encode.SpawnedReplication ( library/Rattletrap/Encode/SpawnedReplication.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/SpawnedReplication.o )
[118 of 227] Compiling Rattletrap.Encode.Int32le ( library/Rattletrap/Encode/Int32le.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Int32le.o )
[119 of 227] Compiling Rattletrap.Encode.Str ( library/Rattletrap/Encode/Str.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Str.o )
[120 of 227] Compiling Rattletrap.Encode.StringAttribute ( library/Rattletrap/Encode/StringAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/StringAttribute.o )
[121 of 227] Compiling Rattletrap.Encode.ReservationAttribute ( library/Rattletrap/Encode/ReservationAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/ReservationAttribute.o )
[122 of 227] Compiling Rattletrap.Encode.PrivateMatchSettingsAttribute ( library/Rattletrap/Encode/PrivateMatchSettingsAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/PrivateMatchSettingsAttribute.o )
[123 of 227] Compiling Rattletrap.Encode.Message ( library/Rattletrap/Encode/Message.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Message.o )
[124 of 227] Compiling Rattletrap.Encode.Mark ( library/Rattletrap/Encode/Mark.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Mark.o )
[125 of 227] Compiling Rattletrap.Encode.Dictionary ( library/Rattletrap/Encode/Dictionary.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Dictionary.o )
[126 of 227] Compiling Rattletrap.Encode.ClassMapping ( library/Rattletrap/Encode/ClassMapping.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/ClassMapping.o )
[127 of 227] Compiling Rattletrap.Encode.StatEventAttribute ( library/Rattletrap/Encode/StatEventAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/StatEventAttribute.o )
[128 of 227] Compiling Rattletrap.Encode.IntAttribute ( library/Rattletrap/Encode/IntAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/IntAttribute.o )
[129 of 227] Compiling Rattletrap.Encode.FlaggedIntAttribute ( library/Rattletrap/Encode/FlaggedIntAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/FlaggedIntAttribute.o )
[130 of 227] Compiling Rattletrap.Encode.ExplosionAttribute ( library/Rattletrap/Encode/ExplosionAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/ExplosionAttribute.o )
[131 of 227] Compiling Rattletrap.Encode.ExtendedExplosionAttribute ( library/Rattletrap/Encode/ExtendedExplosionAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/ExtendedExplosionAttribute.o )
[132 of 227] Compiling Rattletrap.Encode.DamageStateAttribute ( library/Rattletrap/Encode/DamageStateAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/DamageStateAttribute.o )
[133 of 227] Compiling Rattletrap.Encode.AppliedDamageAttribute ( library/Rattletrap/Encode/AppliedDamageAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/AppliedDamageAttribute.o )
[134 of 227] Compiling Rattletrap.Encode.Float32le ( library/Rattletrap/Encode/Float32le.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Float32le.o )
[135 of 227] Compiling Rattletrap.Encode.WeldedInfoAttribute ( library/Rattletrap/Encode/WeldedInfoAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/WeldedInfoAttribute.o )
[136 of 227] Compiling Rattletrap.Encode.PropertyValue ( library/Rattletrap/Encode/PropertyValue.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/PropertyValue.o )
[137 of 227] Compiling Rattletrap.Encode.Property ( library/Rattletrap/Encode/Property.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Property.o )
[138 of 227] Compiling Rattletrap.Encode.Header ( library/Rattletrap/Encode/Header.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Header.o )
[139 of 227] Compiling Rattletrap.Encode.KeyFrame ( library/Rattletrap/Encode/KeyFrame.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/KeyFrame.o )
[140 of 227] Compiling Rattletrap.Encode.FloatAttribute ( library/Rattletrap/Encode/FloatAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/FloatAttribute.o )
[141 of 227] Compiling Rattletrap.Encode.CamSettingsAttribute ( library/Rattletrap/Encode/CamSettingsAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/CamSettingsAttribute.o )
[142 of 227] Compiling Rattletrap.Encode.AttributeValue ( library/Rattletrap/Encode/AttributeValue.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/AttributeValue.o )
[143 of 227] Compiling Rattletrap.Encode.Attribute ( library/Rattletrap/Encode/Attribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Attribute.o )
[144 of 227] Compiling Rattletrap.Encode.UpdatedReplication ( library/Rattletrap/Encode/UpdatedReplication.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/UpdatedReplication.o )
[145 of 227] Compiling Rattletrap.Encode.ReplicationValue ( library/Rattletrap/Encode/ReplicationValue.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/ReplicationValue.o )
[146 of 227] Compiling Rattletrap.Encode.Replication ( library/Rattletrap/Encode/Replication.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Replication.o )
[147 of 227] Compiling Rattletrap.Encode.Frame ( library/Rattletrap/Encode/Frame.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Frame.o )
[148 of 227] Compiling Rattletrap.Encode.Content ( library/Rattletrap/Encode/Content.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Encode/Content.o )
[149 of 227] Compiling Rattletrap.Decode.Common ( library/Rattletrap/Decode/Common.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/Common.o )
[150 of 227] Compiling Rattletrap.Decode.Word8le ( library/Rattletrap/Decode/Word8le.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/Word8le.o )
[151 of 227] Compiling Rattletrap.Decode.Word64le ( library/Rattletrap/Decode/Word64le.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/Word64le.o )
[152 of 227] Compiling Rattletrap.Decode.Word32le ( library/Rattletrap/Decode/Word32le.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/Word32le.o )
[153 of 227] Compiling Rattletrap.Decode.TitleAttribute ( library/Rattletrap/Decode/TitleAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/TitleAttribute.o )
[154 of 227] Compiling Rattletrap.Decode.TeamPaintAttribute ( library/Rattletrap/Decode/TeamPaintAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/TeamPaintAttribute.o )
[155 of 227] Compiling Rattletrap.Decode.QWordAttribute ( library/Rattletrap/Decode/QWordAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/QWordAttribute.o )
[156 of 227] Compiling Rattletrap.Decode.PickupAttribute ( library/Rattletrap/Decode/PickupAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/PickupAttribute.o )
[157 of 227] Compiling Rattletrap.Decode.MusicStingerAttribute ( library/Rattletrap/Decode/MusicStingerAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/MusicStingerAttribute.o )
[158 of 227] Compiling Rattletrap.Decode.LoadoutAttribute ( library/Rattletrap/Decode/LoadoutAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/LoadoutAttribute.o )
[159 of 227] Compiling Rattletrap.Decode.LoadoutsAttribute ( library/Rattletrap/Decode/LoadoutsAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/LoadoutsAttribute.o )
[160 of 227] Compiling Rattletrap.Decode.List ( library/Rattletrap/Decode/List.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/List.o )
[161 of 227] Compiling Rattletrap.Decode.Int8le ( library/Rattletrap/Decode/Int8le.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/Int8le.o )
[162 of 227] Compiling Rattletrap.Decode.Int8Vector ( library/Rattletrap/Decode/Int8Vector.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/Int8Vector.o )
[163 of 227] Compiling Rattletrap.Decode.Int32le ( library/Rattletrap/Decode/Int32le.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/Int32le.o )
[164 of 227] Compiling Rattletrap.Decode.Str ( library/Rattletrap/Decode/Str.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/Str.o )
[165 of 227] Compiling Rattletrap.Decode.StringAttribute ( library/Rattletrap/Decode/StringAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/StringAttribute.o )
[166 of 227] Compiling Rattletrap.Decode.PrivateMatchSettingsAttribute ( library/Rattletrap/Decode/PrivateMatchSettingsAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/PrivateMatchSettingsAttribute.o )
[167 of 227] Compiling Rattletrap.Decode.Message ( library/Rattletrap/Decode/Message.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/Message.o )
[168 of 227] Compiling Rattletrap.Decode.Mark ( library/Rattletrap/Decode/Mark.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/Mark.o )
[169 of 227] Compiling Rattletrap.Decode.StatEventAttribute ( library/Rattletrap/Decode/StatEventAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/StatEventAttribute.o )
[170 of 227] Compiling Rattletrap.Decode.IntAttribute ( library/Rattletrap/Decode/IntAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/IntAttribute.o )
[171 of 227] Compiling Rattletrap.Decode.GameModeAttribute ( library/Rattletrap/Decode/GameModeAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/GameModeAttribute.o )

library/Rattletrap/Decode/GameModeAttribute.hs: warning:
    Could not specialise imported function ‘ghc-prim-0.5.1.1:GHC.Classes.$w$c<1’
      when specialising ‘ghc-prim-0.5.1.1:GHC.Classes.$fOrd(,,)_$c<’
    Probable fix: add INLINABLE pragma on ‘ghc-prim-0.5.1.1:GHC.Classes.$w$c<1’
[172 of 227] Compiling Rattletrap.Decode.Float32le ( library/Rattletrap/Decode/Float32le.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/Float32le.o )
[173 of 227] Compiling Rattletrap.Decode.KeyFrame ( library/Rattletrap/Decode/KeyFrame.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/KeyFrame.o )
[174 of 227] Compiling Rattletrap.Decode.FloatAttribute ( library/Rattletrap/Decode/FloatAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/FloatAttribute.o )
[175 of 227] Compiling Rattletrap.Decode.FlaggedIntAttribute ( library/Rattletrap/Decode/FlaggedIntAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/FlaggedIntAttribute.o )
[176 of 227] Compiling Rattletrap.Decode.EnumAttribute ( library/Rattletrap/Decode/EnumAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/EnumAttribute.o )
[177 of 227] Compiling Rattletrap.Decode.Dictionary ( library/Rattletrap/Decode/Dictionary.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/Dictionary.o )
[178 of 227] Compiling Rattletrap.Decode.PropertyValue ( library/Rattletrap/Decode/PropertyValue.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/PropertyValue.o )
[179 of 227] Compiling Rattletrap.Decode.Property ( library/Rattletrap/Decode/Property.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/Property.o )
[180 of 227] Compiling Rattletrap.Decode.Header ( library/Rattletrap/Decode/Header.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/Header.o )
[181 of 227] Compiling Rattletrap.Decode.DestroyedReplication ( library/Rattletrap/Decode/DestroyedReplication.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/DestroyedReplication.o )
[182 of 227] Compiling Rattletrap.Decode.CompressedWord ( library/Rattletrap/Decode/CompressedWord.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/CompressedWord.o )

library/Rattletrap/Decode/CompressedWord.hs: warning:
    Could not specialise imported function ‘GHC.Float.$w$cproperFraction’
      when specialising ‘GHC.Float.$fRealFracDouble_$cproperFraction’
      when specialising ‘GHC.Float.$fRealFracDouble_$cceiling’
    Probable fix: add INLINABLE pragma on ‘GHC.Float.$w$cproperFraction’
[183 of 227] Compiling Rattletrap.Decode.Vector ( library/Rattletrap/Decode/Vector.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/Vector.o )

library/Rattletrap/Decode/Vector.hs: warning:
    Could not specialise imported function ‘ghc-prim-0.5.1.1:GHC.Classes.$w$c<1’
      when specialising ‘ghc-prim-0.5.1.1:GHC.Classes.$fOrd(,,)_$c<’
    Probable fix: add INLINABLE pragma on ‘ghc-prim-0.5.1.1:GHC.Classes.$w$c<1’
[184 of 227] Compiling Rattletrap.Decode.WeldedInfoAttribute ( library/Rattletrap/Decode/WeldedInfoAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/WeldedInfoAttribute.o )
[185 of 227] Compiling Rattletrap.Decode.LocationAttribute ( library/Rattletrap/Decode/LocationAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/LocationAttribute.o )
[186 of 227] Compiling Rattletrap.Decode.Initialization ( library/Rattletrap/Decode/Initialization.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/Initialization.o )
[187 of 227] Compiling Rattletrap.Decode.SpawnedReplication ( library/Rattletrap/Decode/SpawnedReplication.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/SpawnedReplication.o )

library/Rattletrap/Decode/SpawnedReplication.hs: warning:
    Could not specialise imported function ‘ghc-prim-0.5.1.1:GHC.Classes.$w$c<1’
      when specialising ‘ghc-prim-0.5.1.1:GHC.Classes.$fOrd(,,)_$c<’
    Probable fix: add INLINABLE pragma on ‘ghc-prim-0.5.1.1:GHC.Classes.$w$c<1’
[188 of 227] Compiling Rattletrap.Decode.ExplosionAttribute ( library/Rattletrap/Decode/ExplosionAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/ExplosionAttribute.o )
[189 of 227] Compiling Rattletrap.Decode.ExtendedExplosionAttribute ( library/Rattletrap/Decode/ExtendedExplosionAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/ExtendedExplosionAttribute.o )
[190 of 227] Compiling Rattletrap.Decode.DemolishAttribute ( library/Rattletrap/Decode/DemolishAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/DemolishAttribute.o )
[191 of 227] Compiling Rattletrap.Decode.DamageStateAttribute ( library/Rattletrap/Decode/DamageStateAttribute.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/DamageStateAttribute.o )
[192 of 227] Compiling Rattletrap.Decode.Quaternion ( library/Rattletrap/Decode/Quaternion.hs, /tmp/matrix-worker/1535379711/dist-newstyle/build/x86_64-linux/ghc-8.2.2/rattletrap-5.0.0/build/Rattletrap/Decode/Quaternion.o )

library/Rattletrap/Decode/Quaternion.hs:23:38: error:
    • Variable not in scope: (<>) :: [Char] -> String -> String
    • Perhaps you meant one of these:
        ‘<*>’ (imported from Prelude), ‘*>’ (imported from Prelude),
        ‘>>’ (imported from Prelude)
   |
23 |     y -> fail ("invalid component: " <> show y)
   |                                      ^^
<<ghc: 52071443824 bytes, 1898 GCs, 51354206/135418168 avg/max bytes residency (15 samples), 357M in use, 0.001 INIT (0.001 elapsed), 68.730 MUT (74.855 elapsed), 8.134 GC (8.171 elapsed) :ghc>>
cabal: Failed to build rattletrap-5.0.0 (which is required by exe:rattletrap from rattletrap-5.0.0).
```

I've already revised the release accordingly at https://hackage.haskell.org/package/rattletrap-5.0.0/revisions/ so there's no immediate action required on your part; however, please take this into account for future releases.

Please let me know if something's unclear or you have any questions not covered by the [Hackage Metadata Revisions FAQ](https://github.com/haskell-infra/hackage-trustees/blob/master/revisions-information.md).

--- a/package.yaml
+++ b/package.yaml
@@ -17,7 +17,7 @@ synopsis: Parse and generate Rocket League replays.
 dependencies:
   aeson-pretty: '>= 0.8.5 && < 0.9'
   aeson: '>= 1.2.4 && < 1.5'
-  base: '>= 4.10.1 && < 4.12'
+  base: '>= 4.11.1 && < 4.12'
   binary-bits: '>= 0.5 && < 0.6'
   binary: '>= 0.8.5 && < 0.9'
   bytestring: '>= 0.10.8 && < 0.11'
@@ -25,7 +25,7 @@ dependencies:
   filepath: '>= 1.4.1 && < 1.5'
   http-client-tls: '>= 0.3.5 && < 0.4'
   http-client: '>= 0.5.10 && < 0.6'
-  template-haskell: '>= 2.12.0 && < 2.14'
+  template-haskell: '>= 2.13.0 && < 2.14'
   text: '>= 1.2.3 && < 1.3'
   transformers: '>= 0.5.2 && < 0.6'
 ghc-options:

